pkg/operation/cloudbotanist/alicloudbotanist/controlplane.go

```diff
@@ -129,6 +129,22 @@ func (b *AlicloudBotanist) GenerateKubeSchedulerConfig() (map[string]interface{}
 	return nil, nil
 }
 
+// GenerateETCDStorageClassConfig generates values which are required to create etcd volume storageclass properly.
+func (b *AlicloudBotanist) GenerateETCDStorageClassConfig() map[string]interface{} {
+	return map[string]interface{}{
+		"name":        "etcd-fast",
+		"capacity":    "25Gi",
+		"provisioner": "csi-diskplugin",
+		"parameters": map[string]interface{}{
+			"regionId": b.Shoot.Info.Spec.Cloud.Region,
+			"zoneId":   b.Shoot.Info.Spec.Cloud.Alicloud.Zones[0],
+			"fsType":   "ext4",
```

Comment:
Please change ** fsType** to **csi.storage.k8s.io/fstype**

Reply:
done